### DISCUSSION TITLE
why netbox have 2 redis?

### DISCUSSION THREAD

**Discussion by aryrz** [netbox-community/netbox-docker#727] [General]
hello, 

I am need more info about netbox docker
why you're using 2 redis with different volume? I think its better if netbox docker have a topological picture

**Comment by kkthxbye-code** [comment]
The docker-compose file is optional, you can make your own based on the one in the repo. I'm not sure there is a reason for running two by default though, and I agree that it would probably be better just having one by default, as the redis_cache one does almost nothing after queryset caching was removed.
